docs/dc-apps-performance-toolkit-user-guide-jira.md

```diff
@@ -645,6 +645,48 @@ For more information, go to [Re-indexing Jira](https://confluence.atlassian.com/
 
 Jira will be unavailable for some time during the re-indexing process. When finished, the **Acknowledge** button will be available on the re-indexing page.
 
+---
+
+#### <a id="indexrecovery"></a> Index Recovery (~15 min, only for Jira versions below 9.1.0)
```

Comment:
(~15 min, only for Jira versions 9.0.x and below. For Jira 9.1.0+ skip this step.)